ts/Data/DataTableCore.ts

```diff
@@ -209,10 +212,40 @@ class DataTableCore {
         columnNames?: Array<string>
     ): (DataTable.Row|undefined) {
         return (columnNames || Object.keys(this.columns)).map(
-            (key): DataTableCore.CellType => this.columns[key]?.[rowIndex]
+            (key): DataTable.CellType => this.columns[key]?.[rowIndex]
         );
     }
 
+    /**
+     * Replaces a column reference in the table. It does not do anything else,
+     * unlike the `setColumn` method.
```

Comment:
This description is a bit vague. In order to get it you need to understand how the `setColumn` works.